modules/managed-cluster-notification-view-in-hcc.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa-cluster-notifications.adoc
+// * osd_cluster_admin/osd-cluster-notifications.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="managed-cluster-notification-view-hcc_{context}"]
+= Viewing cluster notifications using Hybrid Cloud Console
+
+Cluster notifications provide important information about the health of your cluster. You can view notifications that have been sent to your cluster in the **Cluster history** tab on Hybrid Cloud Console.
+
+.Prerequisites
+* Your cluster is registered to Hybrid Cloud Console.
```

Comment:
```suggestion
* Your cluster is registered to the Hybrid Cloud Console.
```

Reply:
And throughout this module.

Reply:
All ROSA/OSD clusters are "registered" with OCM. This prereq should be removed.

Reply:
Removed, thanks!